README.md

```diff
@@ -55,11 +57,17 @@ An example of how to use the library
   {#each countries as country}
     <li>{country.name}</li>
   {/each}
-  <SvelteInfiniteScroll threshold={100} on:loadMore={() => page++} />
+  <InfiniteScroll threshold={100} on:loadMore={() => page++} />
 </ul>
 ```
 
-### Another Examples
+In Sapper applications:
```

Comment:
The svelte property in package.json already solves this import issue. If possible, remove this message.

Reply:
Removed as suggested.